src/test/java/Homework17.java

```diff
@@ -35,20 +35,20 @@ public static void AddSongToPlaylist () throws InterruptedException {
         WebElement allSongs = driver.findElement(By.xpath("//a[text()='All Songs']"));
         allSongs.click();
 
-        WebElement song = driver.findElement(By.xpath("//td[text()='Ketsa - That_s a Beat']"));
+        WebElement song = driver.findElement(By.xpath("//td[text()='Mid-Air Machine - If It_s All I Do']"));
         song.click();
 
-        WebElement add = driver.findElement(By.xpath("//button[contains(text(),'Add To…')]"));
+        WebElement add = driver.findElement(By.cssSelector("button[class='btn-add-to']"));
         add.click();
 
         WebElement buttonadd = driver.findElement(By.xpath("//li[contains(text(),'33')]"));
         buttonadd.click();
 
-        WebElement playlist33 = driver.findElement(By.xpath("//a[text()='33']"));
+        WebElement playlist33 = driver.findElement(By.xpath("//*[@id=\"playlists\"]/ul/li[3]/a"));
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

_For example, here is the locator (Copy > Copy Xpath):_  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**
For example, here is the correct locator: ` //a[@class='songs']`

Reply:
Thank you for advises, I know about copy for me problem to find Locators, 80% my time is on that, thank you

Reply:
> Thank you for advises, I know about copy for me problem to find Locators, 80% my time is on that, thank you

Yes, dear!

It is important to mention that the following principle must be held in deciding locators: the locator must be

- unique
- readable 
- maintainable

💖Best wishes, the team HW (Alex)!💖